drivers/include/lis2dh12.h

```diff
@@ -95,6 +95,7 @@ typedef struct {
     i2c_t i2c;                      /**< I2C bus the device is connected to */
     uint8_t addr;                   /**< device address on the I2C bus */
 #endif
+    gpio_t int_pin[2];              /**< Interrupt pins */
```

Comment:
```suggestion
    gpio_t int1_pin;                /**< Interrupt1 pin */
    gpio_t int2_pin;                /**< Interrupt1 pin */
```
Might be easier to use.

Reply:
Then you need an extra if to distinguish them. Otherwise its only one line array wise. I do not know witch one is easier :D

Reply:
Or you move the `gpio_init_int()` inside the `case 1`, `case 2` blocks.